shared/fs/header/header.desktop.js

```diff
@@ -39,7 +39,9 @@ const FolderHeader = ({path, onChat, routePath}: FolderHeaderProps) => (
                 style={styles.headerIcon}
               />
             )}
-            <SendInAppAction path={path} sendIconClassName="" />
+            <Kb.WithTooltip text="Send link">
```

Comment:
Maybe change this to something like “Send to chat”? This will be used for both links and attachments.